pom.xml

```diff
@@ -78,7 +84,7 @@
         </plugins>
     </build>
 
```

Comment:
Add a property for this

```
<properties>
    <artifactory.staging.skip>true</artifactory.staging.skip>
</properties>
```

We don't want to publish this artifact to our staging Artifactory